### DISCUSSION TITLE
force-level1-csd could not find Landsat images

### DISCUSSION THREAD

**Discussion by cxxuuuuuuu** [davidfrantz/force#158] [Q&A] [answered]
Hi all,

I am trying to create base images from 30m Landsat to co-register 10m Sentinel-2. But I couldn't generate any Landsat Level-2 images, as Landsat Level-1 images couldn't be found when I run:

force-level1-csd -d 20150101,20201231 -s LE07,LC08 /home/eouser/src/force/level1 /home/eouser/src/force/level1/level-1-datapool /home/eouser/src/force/level1/l1.pool /home/eouser/src/force/aux/study_area_large.geojson

Querying the metadata catalogue for Landsat data
Sensor(s): LE07,LC08
Tier(s): T1
Tile(s): 160054,160055
Daterange: 20150101 to 20201231
Cloud cover minimum: 0%, maximum: 100%

There were no Landsat Level 1 scenes found matching the search criteria.

Done.

It's super weird, because if I change the sensor types to S2A,S2B, it works and I will have the target images. I have both metadata catalogues of metadata_landsat.csv and metadata_sentinel2.csv under the /home/eouser/src/force/level1/ folder. So I don't understand why I couldn't download the Landsat images.

Have you ever met this situation?

Regards,
Chunyan

**Comment by ernstste** [comment, marked as answer]
Hi, 

there currently is an issue that causes the incorrect Landsat tiles being returned (see #136).
You can manually specify the WRS2 footprints as a workaround (see the docs for force-level1-csd - area of interest - option 3).

The reason why you don't find any Landsat data is probably that the tiles 160054 and 160055 don't cover any land mass.

**Reply by cxxuuuuuuu** [reply]
Hi Stefan,

thank you for your advice. Now I can download the Landsat images by using the right tile number.


Best regards
